radix-engine-interface/src/data/value_serializer.rs

```diff
@@ -940,7 +940,7 @@ mod tests {
                 { "type": "NonFungibleLocalId", "value": "hello" },
                 { "type": "NonFungibleLocalId", "value": "123" },
                 { "type": "NonFungibleLocalId", "value": { "hex": "2345" } },
-                { "type": "NonFungibleLocalId", "value": "371" },
+                { "type": "NonFungibleLocalId", "value": "41635955959434943928015079245517041341" },
```

Comment:
I think I'd rather the value for all of these is just their string encoded form - what do you think?

Reply:
Yeah, we can change to that.